.circleci/config.yml

```diff
@@ -0,0 +1,204 @@
+version: 2.0
+
+jobs:
+  cogutil:
+    docker:
+      - image: opencog/opencog-deps
+        user: root
+        environment:
+          CCACHE_DIR: /ws/ccache
+    working_directory: /ws/cogutil
+    steps:
+      - checkout
+      - run:
+          name: Start restoring ccache
+          command: date +%d-%m-%Y > /tmp/date
+      - restore_cache:
+          keys:
+            - cogutil-ccache-{{ checksum "/tmp/date" }}
+            - cogutil-ccache-
+      - run:
+          name: CMake Configure
+          command: mkdir build && cd build && cmake ..
```

Comment:
Are we going to build and test ```Debug``` or ```Release``` version?
I think ```Release``` can be more appropriate because it is finally should be stable and used by clients.

Reply:
The default is Release

Reply:
Ah, ok, thanks - missed it in CMakeLists.txt.